Favicons added in assets folder

himanshu007-creator opened the pull request:
i have added different sized favicons for varied uses.

himanshu007-creator committed:
Added assets

smaranjitghose commented:
> i have added different sized favicons for varied uses.

Can you please show me a deploy of the code where the favicon is getting integrated this way?

himanshu007-creator commented:
> > i have added different sized favicons for varied uses.
> 
> Can you please show me a deploy of the code where the favicon is getting integrated this way?

sir, i know how to add favicon to an html page, but i dont know how to add to a python file.
Issue was just to add favicon, and that i added. Favicons that i have added are configured according to different requirements. 
if you assign me <a href="https://github.com/smaranjitghose/img_ai_app_boilerplate/issues/10">this Issue</a> then i can work on both side-by-side :+1:

smaranjitghose commented:
> > > i have added different sized favicons for varied uses.
> > 
> > 
> > Can you please show me a deploy of the code where the favicon is getting integrated this way?
> 
> sir, i know how to add favicon to an html page, but i dont know how to add to a python file.
> Issue was just to add favicon, and that i added. Favicons that i have added are configured according to different requirements.
> if you assign me [this Issue](https://github.com/smaranjitghose/img_ai_app_boilerplate/issues/10) then i can work on both side-by-side 👍

Please understand that this a python based project at the core. It is absolutely pointless to do anything related to HTML headers since we are not handling that layer of code and streamlit is.
